### PR TITLE
Trusted clients registry unit tests

### PR DESCRIPTION
**Pull request by emaxx-google**
Add unit tests for the TrustedClientsRegistry
class which is responsible for querying information about the trusted client
application origins from the bundled JSON config.

### CODE REVIEW

**Comment by emaxx-google** [comment]
Time to unveil another 4-year-old pull request :)

File: third_party/pcsc-lite/naclport/server_clients_management/src/permissions_checking/trusted-clients-registry-unittest.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
+/**
+ * @license
+ * Copyright 2019 Google Inc. All Rights Reserved.
```
Comment (Fabian-Sommer) [inline third_party/pcsc-lite/naclport/server_clients_management/src/permissions_checking/trusted-clients-registry-unittest.js:3]: Should we update the year?

**Review by Fabian-Sommer — APPROVED** [review]
LGTM

File: third_party/pcsc-lite/naclport/server_clients_management/src/permissions_checking/trusted-clients-registry-unittest.js (hunk 1 above)
Comment (emaxx-google) [reply third_party/pcsc-lite/naclport/server_clients_management/src/permissions_checking/trusted-clients-registry-unittest.js]: done :laughing:

**Pull request merged**